### DISCUSSION TITLE
Ventura-Infinix Optional Release

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#25] [Discuss about the release]
This is optional Release but final. Experimental.
The END.
But there are some fixes like
- Fixed Our usb is not showing on opencore boot menu.
- Disabled file transfer of USB Type -C charging. For better stability.
- You may face "Volume Hash Mismatch".
- removed previous boot menu UI and Boot chime.
- Removed File Vault.
- downgraded kexts.
- updated some core stuff.
- I suggest you better use this efi than my previous release.
- This EFI works for Ventura, BigSur and Monterey.
- You add previous release EFI device properties to this EFI for better optimisation. And in this efi vram is 3gb so copy all device properties.
- Moved to MacbookAir9,1.
- I am stopping this project due to lack of support.
- No answers will be given and Forking is welcomed. EOL.
# :diamond_shape_with_a_dot_inside:buy me a coffee :coffee:  Donate :  :diamond_shape_with_a_dot_inside:
donate via paypal :
<a href="https://www.paypal.me/369639/"><img src="blue.svg" height="40"></a>  
donate via Paytm:
| <a><img src="darkblue.svg" height="40"></a> |
|-------------------------------------------|
|  `7095160636797@paytm` |

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/'>Ventura-Infinix Optional Release</a>.</em>

**Comment by extasea** [comment]
Does anyone facing the setting apps error?
The setting apps cannot be openned and to fix it we need to restart then reset NVRam. 
Is there any permanent solution besides reset NVRam?
Thanks

**Reply by ghost** [reply]
Have you tried other EFIs yet? Did you get the same problem?

I have not found that issue with me before, but I wish someone else got the resolution for your problem!
Keep us on updated! ‘,:)!

**Reply by devboloji** [reply]
@extasea In which macOS version?

**Reply by extasea** [reply]
@bunnybavros I did, but same

@devboloji Ventura 13.1 the latest one

**Reply by devboloji** [reply]
@extasea  If you are using ventura, use vfinal one. Because your system preferences is crashing due to I mapped USBs in Windows Works well in BigSur and Monterey. So, go with vfinal tagged one, if you are using Ventura.

**Reply by extasea** [reply]
@devboloji yes, I use this final optional EFI
So, should I use the MBP EFI?

**Reply by devboloji** [reply]
@extasea What do you mean by MBP EFI here brother?

**Reply by extasea** [reply]
This EFI https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/vfinal
The SMBIOS has changed from MBA to MBP

**Reply by ghost** [reply]
Did you mean "**MacBook Air**" and "**MacBook Pro**"?

Personally, I kind of used to use "MacBook Air" SMBIOS too, but I see no difference between the air one and the pro one, so I switched to the pro one.

**Reply by devboloji** [reply]
@bunnybavros yup no difference between them. but i prefer to use Air.

**Reply by tushargoyalofficial** [reply]
getting grey screen Infinix X1 Slim

**Reply by tushargoyalofficial** [reply]
tried older release, same issue

**Reply by devboloji** [reply]
@tushargoyalofficial  Replace your new EFI to old EFI and then reset NVRAM once and then boot from USB. IT works.

**Reply by devboloji** [reply]
@keshav200  It's not possible for now. If Opencore team fixes, then I will modify and  release..

**Reply by devboloji** [reply]
@keshav200 yes. If I fix, I will definitely release it.
